AzureAppServiceSetAppSettings/Tasks/AppSettings/ApplyAppSettings.ps1

```diff
@@ -19,8 +22,12 @@ param(
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Internal"
 import-module "Microsoft.TeamFoundation.DistributedTask.Task.Common"
 
+$useSlot = $Slot -ne ""
```

Comment:
instead of doing a useSlot you can replace it by `production` when omitted and make the remaining code more simple.